packages/server-core/src/projects/project/project.service.ts

```diff
@@ -36,6 +37,14 @@ export default (app: Application): void => {
     }
   })
 
+  app.use('project-invalidate', {
+    patch: async ({ projectName }, params) => {
+      if (projectName) {
+        return await useStorageProvider().createInvalidation([`projects/${projectName}*`])
```

Comment:
we need a `/` between `{projectName}` and `*`

Reply:
nevermind, i am wrong! https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Invalidation.html#invalidation-specifying-objects